src/Stratis.Bitcoin.Features.BlockStore/AddressIndexing/AddressIndexer.cs

```diff
@@ -132,15 +136,16 @@ public class AddressIndexer : IAddressIndexer
             this.scriptAddressReader = new ScriptAddressReader();
 
             this.lockObject = new object();
-            this.flushChangesInterval = TimeSpan.FromMinutes(10);
+            this.flushChangesInterval = TimeSpan.FromMinutes(2);
             this.lastFlushTime = DateTime.Now;
             this.cancellation = new CancellationTokenSource();
+            this.chainIndexer = chainIndexer;
             this.logger = loggerFactory.CreateLogger(this.GetType().FullName);
 
             this.averageTimePerBlock = new AverageCalculator(200);
             int maxReorgLength = GetMaxReorgOrFallbackMaxReorg(this.network);
 
-            this.compactionTriggerDistance = maxReorgLength * 2 + SyncBuffer;
+            this.compactionTriggerDistance = maxReorgLength * 2 + SyncBuffer + 1000;
```

Comment:
As a matter of interest, why the additional 1000?

Reply:
So we dont compact data that we will need to rewind in case node is killed during a reorg. So just extra safety.